common/components/add-another/template.njk

```diff
@@ -9,40 +9,44 @@
 {% set itemName = params.itemName or t("item") %}
 {% set items = params.items %}
 
+<div class="moj-add-another">
```

Comment:
I might just be GH's diffs but I can't see a closing tag for this?

Reply:
Nope - I missed it in the various rebase/conflict dances